db/migrate/20151117231542_map_contest_group.rb

```diff
@@ -0,0 +1,11 @@
+class MapContestGroup < ActiveRecord::Migration
+    def change
+        get_all_content_groups = ContestGroup.all()
+
+        get_all_content_groups.each do |c|
+            Contest.where("name LIKE ?" , "%#{c.name}%").update_all({:contest_group_id => c.id})
+        end
+
+        Contest.where("contest_group_id IS NULL").update_all({:contest_group_id => 1})
```

Comment:
Also you can add a NOT NULL constraint to the `contest_group_id`. See `change_column_null` in the Rails docs.